settings/artix7.sh

```diff
@@ -24,4 +24,3 @@ export XRAY_PIN_05="G22"
 export XRAY_PIN_06="F21"
 
 source $(dirname ${BASH_SOURCE[0]})/../utils/environment.sh
-init_db
```

Comment:
this should be also removed from zynq and kintex settings. I think we call this function in some minitests (ROI harness)

Reply:
Done.

I couldn't find `init_db` in any of the minitest directories?

Reply:
I guess we didn't use it there.